src/js/game/tutorial_goals.js

```diff
@@ -11,6 +11,7 @@ export const enumHubGoalRewards = {
     reward_painter: "reward_painter",
     reward_mixer: "reward_mixer",
     reward_stacker: "reward_stacker",
+    reward_unstacker: "reward_unstacker",
```

Comment:
If you add a reward, there also has to be a translation for it. I can add it tho